docs/developers/developer-reference/serialization-format.md

```diff
@@ -156,7 +156,7 @@ Next we encode:
 | `BigUint` | `256`  | `0x0100`           | `0x000000020100` | `256` is the smallest number that takes 2 bytes.                                                             |
 | `BigInt`  | `0`    | `0x`               | `0x00000000`     | Signed `0` is also represented as zero-length bytes.                                                         |
 | `BigInt`  | `1`    | `0x01`             | `0x0000000101`   | Signed `1` is also represented as 1 byte.                                                                    |
-| `BigInt`  | `-1`   | `0x01FF`           | `0x00000001FF`   | The shortest 2's complement representation of `-1` if `FF`. The most significant bit is 1.                   |
+| `BigInt`  | `-1`   | `0xFF`             | `0x00000001FF`   | The shortest 2's complement representation of `-1` is `FF`. The most significant bit is 1.                   |
```

Comment:
Indeed, for top-level encoding, that is the representation.

E.g. https://github.com/multiversx/mx-sdk-js-core/blob/main/src/smartcontracts/codec/binary.spec.ts#L50